cmd/skaffold/app/cmd/flags.go

```diff
@@ -408,12 +409,13 @@ var flagRegistry = []Flag{
 	},
 	{
 		Name:          "add-skaffold-labels",
-		Usage:         "Add Skaffold-specific labels to rendered manifest. If false, custom labels are still applied. Helpful for GitOps model where Skaffold is not the deployer.",
+		Usage:         "Add Skaffold-specific labels to rendered manifest. Custom labels will still be applied. Helpful for GitOps model where Skaffold is not the deployer.",
 		Value:         &opts.AddSkaffoldLabels,
-		DefValue:      true,
+		DefValue:      false,
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"render"},
 		IsEnum:        true,
+		Deprecated:    "Internal Skaffold labels have been deprecated from resources hydrated by Skaffold.",
```

Comment:
These aren't internal labels.
```suggestion
		Deprecated:    "Adding Skaffold-specific labels in `render` is deprecated.",
```